pom.xml

```diff
@@ -15,7 +15,7 @@
   <properties>
     <!-- Just in case the Eclipse project version does not match the parent. -->
     <version.switchyard.runtime>2.1.0.Final</version.switchyard.runtime>
-    <jbtis.version>4.5.0.CR1b</jbtis.version>
+    <jbtis.version>4.5.0.Final</jbtis.version>
```

Comment:
Correct. This is for the final build for SY 2.4.0, which goes with the JBTIS build. I'm sure we'll update the TP for the master branch when we update to SY 2.4.1, which is the next step.